README.md

```diff
@@ -39,3 +39,24 @@ When we took on the task of importing the standards, a few principles guided our
 * **Easy to fix**: Converting 50 state standards is a monumental task (and one we're grateful ASN takes on). There will be typos and we want users to find them and fix them. Thus, each set of standards is versioned and changes can be submitted for approval.
 
 * **Easy to add to**: Schools & districts create their own standards (outside of their state). They should be able to publish those standards and share them with their vendors.
+
+## Development
+
+There are a few tasks that must be completed to get running with a local copy of the project for development purposes:
+
+* Sign up with [Algolia](https://www.algolia.com) and obtain your Application ID and Admin API Key.
```

Comment:
It would be awesome to avoid this. I wonder if we can just stub out the id/key combo or make fake id/keys so the method runs but doesn't actually authenticate.